usb.go

```diff
@@ -163,9 +163,31 @@ func newContextWithImpl(impl libusbIntf) *Context {
 	return ctx
 }
 
-// NewContext returns a new Context instance.
+// NewContext returns a new Context instance with default ContextOptions.
 func NewContext() *Context {
-	return newContextWithImpl(libusbImpl{})
+	return ContextOptions{}.New()
+}
+
+// DeviceDiscovery controls USB device discovery.
+// When set to EnableDeviceDiscovery (default), the connected USB devices will be discovered and enumerated, allowing the use os OpenDevices and OpenWithVIDPID.
+// When set to DisableDeviceDiscovery, OpenDevices will not return any devices. OpenDeviceWithFileDescriptor can be used on some systems to open a device that was otherwise discovered through the operating system.
```

Comment:
trim to 80 chars - a combination of long lines and a multi-line comment doesn't render very well in the side-by-side diff view.